airflow/bin/airflow

```diff
@@ -1,9 +1,30 @@
 #!/usr/bin/env python
 import logging
 import os
+import socket
+import requests
 from airflow import configuration
 from airflow.bin.cli import CLIFactory
 
+def get_private_ip(name=''):
+    r = requests.get("http://169.254.169.254/latest/meta-data/local-ipv4")
```

Comment:
Should "http://169.254.169.254/latest/meta-data/local-ipv4" also be a part of configuration?